### DISCUSSION TITLE
Can I Join Team?

### DISCUSSION THREAD

**Discussion by ImSurajx** [ROM-builders/temporary#10447] [Q&A]
Pixel Experience :)
https://cirrus-ci.com/build/5049382557974528

Pixel Experience Plus :)
https://cirrus-ci.com/build/5891016799551488

ArrowOs Gapps :)
https://cirrus-ci.com/build/6149172117110784

Here Is My Device Tree :)
https://gitlab.com/ImSurajKr/device_xiaomi_lavender

Telegram Username : @ImSurajx09 & @ImSurajx 
GitHuB Username : @ImSurajx 
GitLab Username : @ImSurajKr

**Comment by Apon77** [comment]
> Pixel Experience :) https://cirrus-ci.com/build/5049382557974528
> 
> Pixel Experience Plus :) https://cirrus-ci.com/build/5891016799551488
> 
> ArrowOs Gapps :) https://cirrus-ci.com/build/6149172117110784
> 
> Here Is My Device Tree :) https://gitlab.com/ImSurajKr/device_xiaomi_lavender
> 
> Telegram Username : @ImSurajx09 & @ImSurajx GitHuB Username : @ImSurajx GitLab Username : @Imsurajkr

give some time to mature your github account. also check #learn in telegram group

**Reply by ImSurajx** [reply]
Ok Sir
